scripts/installs/setup_webdav.bat

```diff
@@ -0,0 +1,5 @@
+net stop wampapache
+mkdir C:\wamp\www\uploads\
+copy /Y C:\Vagrant\resources\webdav\httpd.conf C:\wamp\bin\apache\Apache2.2.21\conf\
```

Comment:
Mostly just leaving this here for future reference, but we copy a different httpd.conf (located at resources/wamp/httpd.conf) for the WAMP setup. These two things shouldn't conflict, but I did want to make a note of it.